src/main/java/bio/terra/janitor/common/exception/InvalidMessageException.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.janitor.common.exception;
+
+/**
+ * Exception when {@link bio.terra.generated.model.CreateResourceRequestBody} message is invalid.
+ */
+public class InvalidMessageException extends BadRequestException {
```

Comment:
why does this extend BadRequetException? The pub/sub queue isn't going to report to the ErrorModel of our API.

Reply:
ah, right. RuntimeE?